timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -74,6 +74,11 @@ class AbstractTimeSeriesModel(AbstractModel):
     _preprocess_nonadaptive = None
     _preprocess_set_features = None
 
+    supports_known_covariates: bool = False
+    supports_past_covariates: bool = False
+    supports_cat_covariates: bool = False
+    supports_static_features: bool = False
```

Comment:
This is a GluonTS-specific flag that essentially says "please one-hot encode the categorical covariates", so I don't think it belongs here.

Alternatively, we need to set this to True as well for MLForecast models since they also support cat features.

Reply:
I mean the `supports_cat_covariates` flag